src/elements/content-sidebar/activity-feed/comment/components/BaseCommentMenu.tsx

```diff
@@ -0,0 +1,127 @@
+import * as React from 'react';
+
+import { FormattedMessage } from 'react-intl';
+import TetherComponent from 'react-tether';
+
+import { ACTIVITY_TARGETS } from '../../../../common/interactionTargets';
+import { COMMENT_STATUS_OPEN, COMMENT_STATUS_RESOLVED } from '../../../../../constants';
+import { MenuItem } from '../../../../../components/menu';
+
+import Checkmark16 from '../../../../../icon/line/Checkmark16';
+import DeleteConfirmation from '../../common/delete-confirmation';
+import Media from '../../../../../components/media';
+import messages from '../messages';
+import Pencil16 from '../../../../../icon/line/Pencil16';
+import Trash16 from '../../../../../icon/line/Trash16';
+import X16 from '../../../../../icon/fill/X16';
+
+import { FeedItemStatus } from '../../../../../common/types/feed';
+import './BaseCommentMenu.scss';
```

Comment:
Keep an empty new line between the type imports and the styles imports:
```suggestion
import { FeedItemStatus } from '../../../../../common/types/feed';

import './BaseCommentMenu.scss';
```